### PR TITLE
Point to nightly releases instead of gcr.io

### LINKED ISSUE #343: Fix the link to nightly releases.
# Expected Behavior
Easier link to install nightly releases. Currently in the link for the nightlies, it should point to something that has the release, just nightly, for example:
https://storage.googleapis.com/tekton-releases-nightly/pipeline/latest/release.yaml

# Actual Behavior

```
Additionally, Tekton Pipelines pushes nightly releases every night to gcr.io/tekton-nightly. If you are feeling adventurous and would like to experiment with the most recent, unreleased code, see [Tekton Development Guide](https://github.com/tektoncd/pipeline/blob/main/DEVELOPMENT.md).
```

# Steps to Reproduce the Problem

1.
2.
3.

# Additional Info

### PR DESCRIPTION
**Pull request by vaikas**
Signed-off-by: Ville Aikas <vaikas@chainguard.dev>

# Changes

Point to nightly releases in GCS. Also fix some formatting (trailing whitespace)
Fixes #343 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Review by priyawadhwa — APPROVED** [review]

File: content/en/docs/Getting Started/_index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ description: >
 
 ## Prerequisites
 
-* A Kubernetes cluster version 1.15 or higher for Tekton Pipelines v0.11.0 or higher, or a Kubernetes 
+* A Kubernetes cluster version 1.15 or higher for Tekton Pipelines v0.11.0 or higher, or a Kubernetes
```
Comment (afrittoli) [inline content/en/docs/Getting Started/_index.md:14]: Oh, this is out of date, but we can fix it in a different PR

**Review by afrittoli — COMMENTED** [review]
Thanks!

/approve

**Comment by AlanGreene** [comment]
Looks like @geriom has a PR open to rewrite this content: https://github.com/tektoncd/website/pull/298

We should consider combining the two changes to ensure the nightly builds are still mentioned and more visible.

**Review by vdemeester — APPROVED** [review]
/lgtm

**Pull request merged**